packages/reporter/src/header/header.tsx

```diff
@@ -12,14 +12,16 @@ import { action } from 'mobx'
 import Controls from './controls'
 import Stats from './stats'
 import { StatsStore } from './stats-store'
+import { DebugDismiss } from './DebugDismiss'
 
 export interface ReporterHeaderProps {
   appState: AppState
   events?: Events
   statsStore: StatsStore
+  spec: Cypress.Spec
 }
 
-const Header = observer(({ appState, events = defaultEvents, statsStore }: ReporterHeaderProps) => (
+const Header = observer(({ appState, events = defaultEvents, statsStore, spec }: ReporterHeaderProps) => (
```

Comment:
Not needing to do this now, but we really need to solve App <-> Command Log communication. Although this works for now, I see this becoming a pretty big mess. I don't have any great ideas, but some kind of unified way to bridge the Vue <-> React gap would be great. Let's keep this is mind, definitely open to solutions to this.

Reply:
I looked at the M2 work and there is a lot more communication coming up. We do get reactivity from the `mobxrunnerstore`, maybe we can setup a system whereby any change made to a pinia store is relayed to the mobx store and wrapped with `runInAction`